Dockerfile

```diff
@@ -3,12 +3,16 @@
 #
 
 # Based on docker official node image
-FROM node:5.10.1
+FROM node:8
 
 # Install "nodemon" and "buster" globally
+RUN chown -R node:node /usr/local/lib/node_modules
```

Comment:
This looks really dubious.

Reply:
Encountered some EACCESS error from npm without it, see https://docs.npmjs.com/getting-started/fixing-npm-permissions

Reply:
Having both `USER root` and the `chown` looks useless. Just use `USER root` to install globally? The `chown` are really messing with what the packages install, which is unnecessary and fairly ugly.

Reply:
Found a workaround having the npm globally installed packages in another directory as recommended by npmjs here : https://docs.npmjs.com/getting-started/fixing-npm-permissions (option 2). It seems since npm>5 troubles come when trying to use npm as root.